packages/clay-layout/src/Row.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Vertical positioning of row contents
+	 */
+	align?: 'start' | 'center' | 'end';
+
+	/**
+	 * This removes the negative margins from .row and the
+	 * horizontal padding from all immediate children columns
+	 */
+	gutters?: boolean;
+
+	/**
+	 * Horizontal positioning of row contents
+	 */
+	justify?: 'start' | 'center' | 'end' | 'around' | 'between';
+}
+
+const ClayRow: React.FunctionComponent<IProps> = ({
+	align,
+	children,
+	className,
+	gutters = true,
+	justify,
+	...otherProps
+}) => {
```

Comment:
We didn't add any of these to `<clay:col>` since we couldn't find real usages, so it might be premature API, but LGTM

Reply:
I think its hard for devs to remember the util classes for alignment, my thought was by adding the align and justify props, it would be more accessible to devs via ts autocomplete.